recipe/meta.yaml

```diff
@@ -36,19 +39,22 @@ requirements:
     - pytorch >=1.10.0  # [linux]
```

Comment:
@jnoller @JamesRobertsonGames Could you please check if we should set the [nump](url)y's upper bound `<2.0.0`, see https://github.com/huggingface/accelerate/blame/v0.33.0/setup.py#L73. Or does our numpy 2.0.0 hotfix handle this?

Reply:
@skupr-anaconda I carried that forward from the earlier version so I'm honestly not sure

Reply:
Done in https://github.com/AnacondaRecipes/accelerate-feedstock/pull/5/commits/b1dbebe95febffe68255d882bd61a4501ff1a2f4